SCClassLibrary/Common/Files/ExampleFiles.sc

```diff
@@ -10,4 +10,8 @@ ExampleFiles {
 	*child {
 		^Platform.resourceDir +/+ "sounds" +/+ "a11wlk01.wav";
 	}
+
+	*dir {
+ 		^(Platform.resourceDir +/+ "examples")
+ 	}
```

Comment:
IMO ExampleFiles should be limited static/media files and not code files? At least it feels strange to me that one member of this class is representing something different (code to execute) than the other members of this class (static media assets). I think it would be sufficient to have the code examples under `Platform.exampleDir` - but probably even better under `Path.exampleDir` as the user doesn't need to be aware that such a directory-path is platform-dependent. But this should then wait for https://github.com/supercollider/supercollider/pull/6755